### ISSUE TITLE
[QUESTION] aws-node-termination-handler SQS Queue

### ISSUE THREAD

**Issue by wiseelf** [aws-ia/terraform-aws-eks-blueprints#507]
#### Please describe your quesiton here
aws-node-termination-handler module creates SQS resource as well as cloudwatch event rules resources, and by default configured to work in Queue mode (it has `enableSqsTerminationDraining: true` in values.yaml). What is the reason of this? 
I mean by default after creating it throws a lot of errors:
`2022/05/04 16:25:15 WRN There was a problem monitoring for events error="AWS.SimpleQueueService.NonExistentQueue: The specified queue does not exist for this wsdl version.\n\tstatus code: 400, request id: e7866f99-607d-579d-bd47-3575cb56d5fb" event_type=SQS_TERMINATE`
So basically it doesn't work as expected. Why not to configure queueURL variable in values.yaml?
```
# queueURL Listens for messages on the specified SQS queue URL
# queueURL: ${queueURL}
```
Or maybe I'm missing something and there is a way to configure that?

**Comment by askulkarni2** [comment]
Hi @wiseelf, thanks for bringing this to our attention. This is a bug. You also raise a valid point about forcing Queue mode instead of letting users opt in. We will get this fixed.

**Comment by bryantbiggs** [comment]
@askulkarni2 FYI - https://github.com/aws-samples/eks-blueprints-add-ons/issues/32

**Comment by bryantbiggs** [comment]
I believe this is now resolved with #666
